.changeset/clean-snakes-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `declaration-property-value-no-unknown` parseError even when a file starts with `/* stylelint-disable */`
```

Comment:
[suggest] It seems to me this is a problem of reporting rather than `/* stylelint-disable */`. What about this change (also the PR title)?

```suggestion
Fixed: `declaration-property-value-no-unknown` parse error for `alpha(opacity=n)` to report as violation
```

Reply:
Much better :)